draft-ietf-ace-coap-est-oscore.md

```diff
@@ -279,6 +279,26 @@ In addition, EST-oscore uses the same CoAP Content-Format identifiers when trans
 |       | application/csrattrs                           (res) |  285  |
 {: #table_mediatypes cols="l l" title="EST functions and the associated CoAP Content-Format identifiers"}
 
+Content-Format 281 MUST be supported by EST-oscore servers.
+Servers MAY also support Content-Format 287.
+It is up to the client to support only Content-Format 281, 287 or both.
+As indicated in {{Section 4.3 of RFC9148}}, the client will use a CoAP Accept Option in the request to express the preferred response Content-Format.
+If an Accept Option is not included in the request, the client is not expressing any preference and the server SHOULD choose format 281.
+
+The generated response for /skg and /skc requests contains two parts: certificate and the corresponding private key.
+{{Section 4.8 of RFC9148}} specifies that the private key in response to /skc request may be either an encrypted (PKCS #7) or unencrypted (PKCS #8) key, depending on whether the CSR request included SMIMECapabilities.
+
+Due to the use of OSCORE, which protects the communication between the EST client and the EST server end-to-end, it is possible to return the private key to /skc as an unencrypted PKCS #8 object (Content-Format identifier 284).
+Therefore, when making the CSR to /skc, the EST client MUST NOT include SMIMECapabilities.
+As a consequence, the response to /skc is an unencrypted PKCS #8 object.
```

Comment:
Isn't the unencrypted PKCS #8 object only the private key? The second item is pkix-cert.

Reply:
Rephrased this to "the private key part of the response to /skc is an unencrypted PKCS #8 object".